### ISSUE TITLE
[Proposal] Create a markdown listing of cloud native security gaps

### ISSUE THREAD

**Issue by TheFoxAtWork** [cncf/sig-security#598]
Description: need to create an index or listing of gaps in tech or tooling for cloud native security so as we trip over them they are annotated but so we may also refer to the list as opportunities for discovery and inclusion into the CNCF

Impact: this transitions tribal knowledge from the SIG into public visibility as known areas for improvement without a current solution space

Scope: Initially a data call across existing projects to discover what we've tripped over already, and then the PR to add it.

TO DO
- [ ] SIG Representative
- [ ] Project leader(s)
- [ ] TBD

**Comment by rficcaglia** [comment]
is this different from the CNSMAP effort?

**Comment by TheFoxAtWork** [comment]
Yes!  while linked with the CNSMAP effort (much of those activities will drive this content) there are other areas (ideas, discoveries, community slack discussions) that reveal other gaps.  Not just in tech and tooling, but in community needs, understanding, etc.

**Comment by apmarshall** [comment]
I am imagining this as something like a roadmap with upvoting: anyone can submit an idea for where there is a "gap"/pain-point, the community can vote/like/react to ideas to "second" that something is indeed a gap/pain-point, and there can be comments/discussions proposing possible solutions (whether those be existing products that weren't widely known or ideas for creating a new project to fill a gap). Does that vision fit with what others are imagining for this?
